samples/Tiny_MeshLarge.cpp

```diff
@@ -1729,9 +1730,7 @@ lvk::TextureHandle createTexture(const LoadedImage& img) {
       },
       nullptr);
 
-  if (!hasCompressedTexture) {
-    ctx_->generateMipmap(tex);
-  }
+  ctx_->generateMipmap(tex);
```

Comment:
This will destroy preloaded mipmaps. Should be called only when no mipmaps are loaded from `.ktx` files.